test/pages/chat/events/message/message_test.dart

```diff
@@ -0,0 +1,120 @@
+// ignore_for_file: depend_on_referenced_packages
+import 'package:fluffychat/config/localizations/localization_service.dart';
+import 'package:fluffychat/domain/usecase/preview_url/get_preview_url_interactor.dart';
+import 'package:fluffychat/utils/responsive/responsive_utils.dart';
+import 'package:fluffychat/widgets/twake_app.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:flutter_localized_locales/flutter_localized_locales.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:fluffychat/pages/chat/events/message/message.dart';
+import 'package:fluffychat/widgets/avatar/avatar.dart';
+import 'package:get_it/get_it.dart';
+import 'package:inview_notifier_list/inview_notifier_list.dart';
+import 'package:matrix/matrix.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import '../../../../utils/test_client.dart';
+import 'message_test.mocks.dart';
+
+class MockTimeline extends Mock implements Timeline {}
+
+@GenerateNiceMocks([
+  MockSpec<Room>(),
+  MockSpec<Event>(),
+  MockSpec<Client>(),
+  MockSpec<ResponsiveUtils>(),
+  MockSpec<BuildContext>(),
+  MockSpec<GetPreviewURLInteractor>(),
+])
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  late MockTimeline mockTimeline;
+  late Event event;
+  late Event nextEvent;
+  late Event previousEvent;
+  late Room room;
+  late Client client;
+  late GetPreviewURLInteractor previewURLInteractor;
+
+  setUpAll(() async {
+    previewURLInteractor = MockGetPreviewURLInteractor();
+    final getIt = GetIt.instance;
+    getIt.registerSingleton(ResponsiveUtils());
+    getIt.registerSingleton(previewURLInteractor);
+    client = await prepareTestClient(loggedIn: true);
+  });
+
+  group('Message Widget Tests', () {
+    setUp(() {
+      room = MockRoom();
+      mockTimeline = MockTimeline();
+      event = MockEvent();
+      previousEvent = MockEvent();
+      nextEvent = MockEvent();
+      // client = MockClient();
+    });
```

Comment:
okie, I think we should go simple in this situation. Unit test for small method to avoid mock a lot of things.